scss/_spinners.scss

```diff
@@ -1,36 +1,45 @@
 //
-// Rotating border
+// Spinners
 //
 
-@keyframes spinner-border {
-  to { transform: rotate(360deg); }
-}
-
-.spinner-border {
+%spinner {
   position: relative;
   display: inline-block;
   width: $spinner-width;
   height: $spinner-height;
   overflow: hidden;
   text-indent: -999em;
   vertical-align: text-bottom;
+}
+
+//
+// Rotating border
+//
+
+@keyframes rotate {
+  to { transform: rotate(360deg); }
+}
+
+.spinner-border {
+  @extend %spinner;
   border: $spinner-border-width solid;
   border-color: currentColor transparent currentColor currentColor;
   border-radius: 50%;
-  animation: spinner-border .75s linear infinite;
-}
+  animation: rotate .75s linear infinite;
 
-.spinner-border-sm {
-  width: $spinner-width-sm;
-  height: $spinner-height-sm;
-  border-width: $spinner-border-width-sm;
+  &.spinner-sm {
```

Comment:
I would not use the `&` here, we don't do that for other modifiers, eg `.btn-lg`: https://github.com/twbs/bootstrap/blob/d96c147d6715b4772608f5d4b3ddd9cb0c7af7a3/scss/_buttons.scss#L110

Reply:
Also, the class is not correct